sdk/tests/events/changelog.test.ts

```
@@ -1,100 +1,116 @@
 
-            // Get noop program instruction
-            const accountKeys = transaction!.transaction.message.getAccountKeys();
-            const noopInstruction = transaction!.meta!.innerInstructions![0].instructions[0];
-            const programId = accountKeys.get(noopInstruction.programIdIndex)!;
-            if (!programId.equals(MPL_NOOP_PROGRAM_ID)) {
-                throw Error(`Only inner ix should be a noop, but instead is a ${programId.toBase58()}`);
-            }
-            const cpiData = Buffer.from(bs58.decode(noopInstruction.data));
-            const changeLogEvent = deserializeChangeLogEventV1(cpiData);
+      // Get noop program instruction
+      const accountKeys = transaction!.transaction.message.getAccountKeys();
```

Biome (1.9.4)

[error] Forbidden non-null assertion.

Unsafe fix: Replace with optional chain operator ?. This operator includes runtime checks, so it is safer than the compile-only non-null assertion operator


(lint/style/noNonNullAssertion)